config/wpcalypso.json

```diff
@@ -123,6 +123,7 @@
 		"press-this": true,
 		"privacy-policy": true,
 		"publicize-preview": true,
+		"push-notifications": true,
```

Comment:
Any reason we didn't have this enabled here before?

Reply:
I think it was due to the way we only store a single browser token on the server-side, so if you switched between wpcalypso and production, your token would rotate frequently between the two, and your notifications would be delivered from either wpcalypso or production, depending on which you visited last. It wasn't a problem for development, because you have to jump through hoops to get it to work there... but since calypso.live is based on wpcalypso, I had to enable it there for testing -- I can potentially disable it again before merging. Thoughts?

Reply:
Hmm... probably fine to keep enabled. Either way is potentially confusing.

Reply:
Actually -- I think I might disable it because of calypso.live. If your last token was from a .live instance that went away, your notifications will break.

Reply:
Oh, great point -- agree!